### DISCUSSION TITLE
Feature request. H2-10K-T2

### DISCUSSION THREAD

**Discussion by Klagio** [djansen1987/SAJeSolar#31] [General]
Hi just received two of these hybrid inverters, together with batteries B1-5.1-48V and METER CHINT SMART DDSU666.

Any advice on how to integrate it?

**Comment by djansen1987** [comment]
Look very nice! Adding support for a new unit will need some investigation with access to an account with this device in it. But first have you tried the H1 setup to see if this works ?

**Reply by djansen1987** [reply]
were you able to add the device ?
